components/brave_rewards/browser/rewards_service.cc

```diff
@@ -26,6 +28,15 @@ void RewardsService::RemoveObserver(RewardsServiceObserver* observer) {
   observers_.RemoveObserver(observer);
 }
 
+// static
+bool RewardsService::IsSupported(PrefService* prefs) {
+  if (IsDisabledByPolicy(prefs) || IsDisabledByFeature()) {
```

Comment:
`return !IsDisabledByPolicy(prefs) && !IsDisabledByFeature();`?

Reply:
Changed in https://github.com/brave/brave-core/pull/14928/commits/8bfd36f672940008123eda01889f44d90d14e7c2